src/designer/elsa-workflows-studio/src/components/screens/workflow-instance-viewer/elsa-workflow-instance-viewer-screen/elsa-workflow-instance-viewer-screen.tsx

```diff
@@ -339,6 +385,21 @@ export class ElsaWorkflowInstanceViewerScreen {
                             onActivitySelected={e => this.onActivitySelected(e)}
                             onActivityDeselected={e => this.onActivityDeselected(e)}
                             onActivityContextMenuButtonClicked={e => this.onActivityContextMenuButtonClicked(e)}
+        /> */}
+        <elsa-designer
+          model={this.workflowModel}
+          mode={WorkflowDesignerMode.Instance}
+          layoutDirection={this.layoutDirection}
+          activityContextMenuButton={this.renderActivityStatsButtonTest}
+          activityBorderColor={this.getActivityBorderColor}
+          onActivityContextMenuButtonTestClicked={e => this.onActivityContextMenuButtonClicked(e)}
+          // !!!! onActivityContextMenuButtonClicked={e => this.onActivityContextMenuButtonClicked(e)}
+          // !!!! onActivityContextMenuButtonTestClicked={e => this.onActivityContextMenuButtonTestClicked(e)}
+          activityContextMenu={this.activityContextMenuState}
+          onActivitySelected={e => this.onActivitySelected(e)}
+          onActivityDeselected={e => this.onActivityDeselected(e)}
+          class="canvas-container"
+          ref={el => (this.designer = el)}
         />
```

Comment:
Same as previous comment about hardcoding the new designer component in - this needs to be configurable.